### ISSUE TITLE
Step 8 Scale with Cloud SQL

### ISSUE THREAD

**Issue by Wasabi95** [googlecodelabs/full-stack-store-locator#4]
I followed all the steps but I got stuck in step number 8 Scale with Cloud SQL when creating file " locations.go " error error loading workspace you are outside of a module.  what might be the mistake?
![2021-12-31](https://user-images.githubusercontent.com/74787751/147834901-c34f8f05-512e-487b-a61b-5ea8f3772506.png)

**Comment by AtomicSpirit** [comment]
I am stuck on step 8 as well.  Up to the point i'm getting warnings from google cloud for the cost of my usage LOL

**Comment by Wasabi95** [comment]
yes, and there is not the only lab in Codelabs with issues, I have tried
many times different labs and I could not complete them.

**Comment by jpoehnelt** [comment]
Should update example to use https://go.dev/blog/using-go-modules.

**Comment by slawson612** [comment]
Yep stuck on step 8. The issues I am hitting are two - the first issue is a 404 with the dropoffs function handler and a JSON p token on the response. I have confirmed that the data *IS* in the db with a simple script that returns the first 25 records so I know the are values out there. Has anyone on this thread got this to work yet?

As a side note: I wanted to be sure I had not fat fingered something so I DLed the full package from gitHub and pasted all the provided code into the interface, into the appropriate scripts and got the same two errors.

**Comment by dspenard** [comment]
I, too, was getting stuck on Step 8, although it was due to a 503 error on the call to **/data/dropoffs** to fetchStores.  My issue was a dumb fat finger of **export DB_TCP_HOST=127.0.01**, instead of 127.0.0.1.  That fixed the issue, so I suggest just going slowly back over step 8 to see what is missed either with the database setup, env vars, or the code, as step 8 is working.

**Comment by slawson612** [comment]
Thanks David.

I went back and did a straight cut and paste from the source code on
github.

I would not be surprised if it is something I did but it is the same error
whether it is my fingers or theirs as far as I can tell.

Steve

**Comment by dspenard** [comment]
Glad to hear it worked, Steve.  Although I did have issue with deploying it to App Engine today, as the Go version used in CloudShell for the sample is incompatible with App Engine Flexible.  I didn't have time to investigate but I plan on building and deploying from my local machine anyway, so I'll address it then.  But it is annoying when example walkthroughs like this are not kept up to date.

Update:  It was actually a simple thing, I needed to specify **runtime: go1.15** in app.yaml and **go 1.15** in go.mod and I was able to deploy to App Engine Flexible.  Additionally, I used my own service account and not the default, so Step 11 is tweaked a little in that case.
